src/lib/styles/themes/dark.scss

```diff
@@ -10,6 +10,10 @@
   --text-text: var(--purple-900);
   --input-focus-background: var(--purple-dark-450);
   --button-primary: var(--purple-dark-accent);
+  --table-background: var(--purple-dark-550);
+  --table-header-background: var(--purple-dark-450);
+  --table-row-hover: var(--purple-dark-500);
+  --table-row: var(--purple-dark-550);
```

Comment:
Are these last 2 also backgrounds? Should we not add `background` to their names as well?

Reply:
These are the same name as in Figma, but let me ask Artem to change it. It makes sense to me.